metadata-ingestion/src/datahub/integrations/great_expectations/action.py

```diff
@@ -317,9 +317,15 @@ def get_assertions_with_results(
                     type=AssertionResultType.SUCCESS
                     if success
                     else AssertionResultType.FAILURE,
-                    rowCount=result.get("element_count"),
-                    missingCount=result.get("missing_count"),
-                    unexpectedCount=result.get("unexpected_count"),
+                    rowCount=None
```

Comment:
Can we add a `_int_or_none` helper method that does the casting logic - not a huge can of the repetition here

Reply:
I have optimized these codes, thank you for your suggestion.